compiler/vmgen.nim

```diff
@@ -1317,11 +1320,11 @@ proc genMagic(c: PCtx; n: PNode; dest: var TDest; m: TMagic) =
     else:
       globalError(c.config, n.info, "expandToAst requires a call expression")
   of mSizeOf:
-    globalError(c.config, n.info, "cannot evaluate 'sizeof' because its type is not defined completely")
+    globalError(c.config, n.info, sizeOfLikeMsg("sizeof"))
   of mAlignOf:
-    globalError(c.config, n.info, "cannot evaluate 'alignof' because its type is not defined completely")
+    globalError(c.config, n.info, sizeOfLikeMsg("alignof"))
   of mOffsetOf:
-    globalError(c.config, n.info, "cannot evaluate 'offsetof' because its type is not defined completely")
+    globalError(c.config, n.info, sizeOfLikeMsg("offsetof"))
```

Comment:
I know you try to normalize the error message here. But I prefer the denormalized code here. In my workflow I often grep for error messages in the Nim compiler, and that works better if the error message is contained in a string literal in the call to `globalError` instead of a template. Also I think a call to globalError with an error message string literal is easier to read than this shorter notation.

Reply:
fixed more generally via https://github.com/nim-lang/Nim/pull/14317, no more grepping / guess work needed with `--hint:msgOrigin`